### PR TITLE
684 issue no fixed

### PR DESCRIPTION
**Pull request by shalishi**
Add alert when user choose similar font color and txt color.
Please let me know if there is any problem.
Thank you.

# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
If there are any UI change, please include the screenshots also.

Fixes #(issue) #684

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: app/src/main/java/swati4star/createpdf/fragment/TextToPdfFragment.java [hunk 1]
```diff
@@ -257,6 +266,16 @@ private void setFontColor() {
...
+                    int alpha = Color.alpha(mFontColor);
+                    mFontColorRed = Color.red(mFontColor);
+                    mFontColorBlue = Color.blue(mFontColor);
+                    mFontColorGreen = Color.green(mFontColor);
+                    double colorDif = Math.sqrt((Math.pow((mFontColorRed - mPageColorRed), 2) +
+                            Math.pow((mFontColorBlue - mPageColorBlue), 2)
+                            + Math.pow((mFontColorGreen - mPageColorGreen), 2)));
+                    if (colorDif < 30) {
+                        showSnackbar(mActivity, R.string.snackbar_color_too_close);
+                    }
```
[the remark this answers is not in the document]
Comment (shalishi) [reply app/src/main/java/swati4star/createpdf/fragment/TextToPdfFragment.java]: Thanks, those two parts were just corrected

[the remark this answers is not in the document]
Comment (shalishi) [reply app/src/main/java/swati4star/createpdf/fragment/TextToPdfFragment.java]: *three parts*

**Comment by vyankatesh24** [comment]
> Add alert when user choose similar font color and txt color.
> Please let me know if there is any problem.
> Thank you.
> 
> # Description
> Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
> If there are any UI change, please include the screenshots also.
> 
> Fixes #(issue)
> 
> ## Type of change
> Just put an x in the [] which are valid.
> 
> * [ ]  Bug fix (non-breaking change which fixes an issue)
> * [ ]  New feature (non-breaking change which adds functionality)
> 
> # How Has This Been Tested?
> Please describe the tests that you ran to verify your changes.
> 
> * [ ]  `./gradlew assembleDebug assembleRelease`
> * [ ]  `./gradlew checkstyle`
> 
> # Checklist:
> * [ ]  My code follows the style guidelines of this project
> * [ ]  I have performed a self-review of my own code
> * [ ]  I have commented my code, particularly in hard-to-understand areas
> * [ ]  I have made corresponding changes to the documentation
> * [ ]  My changes generate no new warnings

@shalishi 
Add issue no in front of `Fixes #(issue)` and put x in [ ] wherever applies

File: app/src/main/res/values-ru/strings.xml [hunk 2]
```diff
@@ -71,6 +71,9 @@
     <string name="snackbar_file_not_deleted">Невозможно удалить файл.</string>
     <string name="snackbar_folder_not_created">При создании директории произошла ошибка.</string>
     <string name="pdf_merged">Файлы PDF объединены</string>
+    <string name="snackbar_color_too_close">Ваш цвет шрифта и цвет текста слишком похожи</string>
+
```
Comment (vyankatesh24) [inline app/src/main/res/values-ru/strings.xml]: Remove these extra blank lines.

**Comment by vyankatesh24** [comment]
@shalishi 
Just do these changes, Everything else seems fine :+1:

File: app/src/main/res/values-ru/strings.xml (hunk 2 above)
Comment (shalishi) [reply app/src/main/res/values-ru/strings.xml]: It's done. Thanks for reviews

**Comment by shalishi** [comment]
Thanks you guys^^

File: app/src/main/java/swati4star/createpdf/fragment/TextToPdfFragment.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (shalishi) [reply app/src/main/java/swati4star/createpdf/fragment/TextToPdfFragment.java]: Thanks a lot for your suggestion and help. I'll do better in the future ^^

**Comment by vyankatesh24** [comment]
Congratulations @shalishi on your first contribution to `Images-to-PDF` :+1:

**Pull request merged**